custom_components/powercalc/data/signify/929002469202/model.json

```
@@ -0,0 +1,14 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "PeakTech P 4250 (1 mV/10 mA) with VOLTCRAFT VC270",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 1,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.10"
+    },
+    "name": "Signify (Hue) 929002469202",
```

```suggestion
    "name": "Hue white A60 bulb E27 1050lm with Bluetooth",
```

This needs to be the full marketed name of the light, something like this.. Will be displayed in the supported model list.

---

Got it, using the name from zigbee2mqtt 👍🏼😊

---

Thanks for the contribution. Everything seems good to merge now.